www/shared/test/utils/render-with-intl/index.js

```diff
@@ -0,0 +1,2 @@
+export { default as renderWithIntl } from './render-with-intl';
```

Comment:
This export is unnecessary. However, we will remove this file altogether, please continue reading.